### DISCUSSION TITLE
ch5-web/ch5-08-interface-and-web

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#632] [General]
# ch5-web/ch5-08-interface-and-web

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch5-web/ch5-08-interface-and-web.html

**Comment by dingdinglz** [comment]
表驱动似乎要创建多个对象？当我可能只需要一个的场景是不是不如switch了

**Reply by cch123** [reply]
> 表驱动似乎要创建多个对象？当我可能只需要一个的场景是不是不如switch了

一个场景不需要切换实现，那怎么简单怎么来，简单的 step1 step2 step3 封装就够了
